components/remote_settings/src/macros.rs

```diff
@@ -0,0 +1,41 @@
+#[macro_export]
+macro_rules! packaged_collections {
+   ($($collection:expr),* $(,)?) => {
+       fn get_packaged_data(collection_name: &str) -> Option<&'static str> {
+           match collection_name {
+               $($collection => Some(include_str!(concat!(
+                   env!("CARGO_MANIFEST_DIR"),
+                   "/dumps/main/",
+                   $collection,
+                   ".json"
+               ))),)*
+               _ => None,
+           }
+       }
+   };
+}
+
+#[macro_export]
+macro_rules! packaged_attachments {
+    () => {
```

Comment:
I left this in, since I am not sure if we keep the `regions` collections + attachments in tree. Can update the PR once we decide.

Reply:
This example data is great because it almost never changes, but it's also kind of big. 
Search config icons are smaller in size, and will eventually be needed AFAIU, but I admit that the list is kind of long and not so elegant to serve as an example